src/com_tjvendors/site/views/vendor/view.html.php

```diff
@@ -85,6 +88,13 @@ public function display($tpl = null)
 		Text::script('COM_TJVENDOR_DESCRIPTION_READ_MORE');
 		Text::script('COM_TJVENDOR_DESCRIPTION_READ_LESS');
 
+		$this->isAdmin = 0;
+		
+		if ($app->isAdmin())
+		{
+			$this->isAdmin = 1;
```

Comment:
@deepgawade89 is admin itself return true or false. Why declared 0 or 1 like this

Reply:
Resolved